priv/templates/extended_bin

```diff
@@ -64,7 +63,7 @@ relx_rem_sh() {
 
 # Generate a random id
 relx_gen_id() {
-    od -t x -N 4 /dev/urandom | head -n1 | awk '{print $2}'
+    od -X -N 4 /dev/urandom | head -n1 | gawk '{print $2}'
```

Comment:
It's not needed in this particular line, but may be needed in other place (see comments below), so this one has been renamed to match other gawk invocations.